test/test_pydantic_class_generation.py

```diff
@@ -0,0 +1,55 @@
+import scanpy as sc
+
+from biochatter.api_agent.generate_pydantic_classes_from_module import generate_pydantic_classes
+
+
+def test_generate_pydantic_classes() -> None:
```

Comment:
@slobentanzer is typing relevant for unit functional tests?

Reply:
no, this can be deactivated